lib/src/cli.rs

```diff
@@ -907,6 +907,9 @@ impl Opt {
                 InternalsOpts::GENERATOR_BIN => {
                     Some(["bootc", "internals", "systemd-generator"].as_slice())
                 }
+                "ostree-container" | "ostree-ima-sign" | "ostree-provisional-repair" => {
+                    Some(["bootc", "internals", "ostree-ext"].as_slice())
```

Comment:
(thinking out loud)

My brain is saying "...you dropped the verb here" but actually it gets in via the `.chain()` below taking the entire argument list again. not just argv[1..].  But that means that we also include the `::GENERATOR_BIN` part (without path mangling) after the `systemd-generator` argument.   Which is presumably correct because that's how it already works today....

Reply:
> My brain is saying "...you dropped the verb here" but actually it gets in via the .chain() below taking the entire argument list again.

Not quite, the `next()` from the iterator popped off the first arg.

I pushed another commit "cli: Strengthen generator parsing test" which shows the existing code was right.